.github/workflows/dispatch.yml

```diff
@@ -0,0 +1,41 @@
+# Trigger the execution of copy-workflow.yml, in batches of 50 repositories.
+# This workflow is needed since GitHub Actions limits the matrix size to 256 jobs.
+# We use one job per repository per file.
+
+on:
+  pull_request:
+    branches: [ master ]
+    types: [ closed ] # IMPORTANT: check the merge status for every job: github.event.pull_request.merged == true
+
+env:
+  MAX_REPOS_PER_WORKFLOW: 50
+  WORKFLOWS: '[ "autorebase", "automerge", "go-test", "go-check" ]' # a JSON array of the files to distribute
+
+jobs:
+  matrix:
+    if: github.event.pull_request.merged == true
+    name: Trigger copy workflows
+    runs-on: ubuntu-latest
+    outputs:
+      targets: ${{ steps.set-matrix.outputs.targets }}
+    steps:
+      - uses: actions/checkout@v2
+      - id: set-matrix
+        run: |
+          TARGETS=$(jq '. | _nwise(${{ env.MAX_REPOS_PER_WORKFLOW }})' .github/workflows/config.json | jq -sc '. | to_entries')
+          echo "::set-output name=targets::$TARGETS"
+  dispatch:
+    if: github.event.pull_request.merged == true
+    needs: [ matrix ]
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        cfg: ${{ fromJson(needs.matrix.outputs.targets) }}
+    name: Start copy workflow (batch ${{ matrix.cfg.key }})
+    steps:
+      - uses: peter-evans/repository-dispatch@ce5485de42c9b2622d2ed064be479e8ed65e76f4 # v1.1.3
```

Comment:
is there a github forum thread or issue about doing this without a third party action? if so, it would be good to add it as a TODO comment, so we use more standard features longer-term

Reply:
I haven't seen any. As far as I can tell, GitHub seems to be happy if things can be solved using 3rd party actions.